README.md

```diff
@@ -19,7 +19,7 @@ Then, pull the submodule with data loaders and install its code:
 
```

Comment:
please update branch and resolve conflicts

Reply:
done